### PR TITLE
Add PrivateIP option for cloning Linodes

### PR DESCRIPTION
**Pull request by zliang-akamai**
Linode API v4 now supports having a new linode created from cloning being assigned a private net IP. Adding corresponding feature in linodego.

### CODE REVIEW

**Review by jriddle-linode — APPROVED** [review]

File: test/integration/instances_test.go [hunk 1]
```diff
@@ -375,6 +375,54 @@ func TestInstance_Rebuild(t *testing.T) {
...
+		*cloned_instance.Created,
+		240,
+	)
+
+	if err != nil {
+		t.Error(err)
+	}
+
+	// clean up
+	client.DeleteInstance(context.Background(), cloned_instance.ID)
```
Comment (lgarber-akamai) [inline test/integration/instances_test.go:423]: Could we add a step to validate that the cloned instance has a private IP, etc?

**Review by lgarber-akamai — CHANGES_REQUESTED** [review]
Looks great! Just one suggestion

File: test/integration/instances_test.go (hunk 1 above)
Comment (zliang-akamai) [reply test/integration/instances_test.go:440]: Added, thank you!

**Comment by lgarber-akamai** [comment]
/acctest sha=8626ee6a68d707d638f802e17d176335f4fedb26 tests=TestInstance_Clone

**Review by lgarber-akamai — APPROVED** [review]

**Pull request merged**